### PR TITLE
Added Force scan to NZBGet

### PR DESCRIPTION
**Pull request by jarlave**
With this button you can force scan the incoming nzb directory

I hope that I implemented it correctly.
For me it works.

### CODE REVIEW

File: interfaces/default/js/nzbget.js [hunk 1]
```diff
@@ -88,7 +88,16 @@ $(document).ready(function () {
 
     });
 
-
+    $('#nzb_force_scan').click(function(e) {
+        e.preventDefault();
+        var speed = ($('#nzb_get_speed').val());
```
Comment (Hellowlol) [inline interfaces/default/js/nzbget.js]: What is that line used for?

**Comment by jarlave** [comment]
You mean the nzb_get_speed? I see, I make a mistake there, forgot to remove this, I copied it from an other block... sry, my first time

File: interfaces/default/html/nzbget.html [hunk 2]
```diff
@@ -17,6 +17,7 @@ <h1 class="page-header page-title">
             <!-- Fake click as type=file is ugly in boostrap -->
             <a class="btn btn-info" onclick="$('#add_nzb_file').click();">Browse</a>
             <input id="add_nzb_file" style="display:none;" type="file">
+            <button class="btn btn-warning" id="nzb_force_scan">Force Scan</button>
```
Comment (Hellowlol) [inline interfaces/default/html/nzbget.html:20]: Would it be better with a icon?

**Comment by Hellowlol** [comment]
Merged, Thanks!

**Pull request closed without merging**